angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts

```diff
@@ -48,35 +73,260 @@ is 54rem (864px). If the content needs of your application
 require this breakpoint to be larger or smaller, it can be
 changed in your application’s Sass file with the
 variable \`$sprk-masthead-breakpoint\`.
-- In Default Masthead the little nav is the section to the
+- In the Default Masthead the Navigation Items section is to the
```

Comment:
I'm having trouble understanding this sentence because of how this is written.

I think the heart of what this is saying is: 
`The Default Masthead allows for one row of navigational items. If there are too many to fit in one row, use Extended Masthead instead because it allows for two rows of navigational items.`

Reply:
yeah that is right! I'll update it with that.